src/energyplus/ReverseTranslator.cpp

```diff
@@ -880,6 +880,10 @@ namespace energyplus {
         modelObject = translateSizingPeriodDesignDay(workspaceObject);
         break;
       }
+      case openstudio::IddObjectType::SolarCollectorPerformance_PhotovoltaicThermal_BIPVT: {
+        modelObject = translateSolarCollectorPerformancePhotovoltaicThermalBIPVT(workspaceObject);
+        break;
+      }
```

Comment:
I'm adding the RT for the future mostly, since the SolarCollectorPVT isn't RT'ed (since PlantLoop isn't).

But this can still be useful if you want to grab the Perf data from an IDF to assign to a model-time created SolarCollectorPVT.